database/insertData/04_users.js

```diff
@@ -34,12 +28,6 @@ exports.queries = [
           passwordHash: "$2a$10$3Z1cXVI.GzE9F2QYePzbMOg5CGtf6VnNKRiaiRGkzlBXJ0aiMN4JG"
           username: "carl"
           firstName: "Carl", lastName: "Smith"
-          permissionJoinsUsingId: { 
-            create: [
-              # Self assign
-              { permissionNameId: 10500 }
-            ]
-          }
```

Comment:
Hey, just FYI -- I have gone an done a fairly big update to all these in #303, so maybe avoid too many changes to the templates that aren't directly related to the PR's functionality, otherwise it's going to be messy to fix.

Reply:
Yes, that's a good idea. I just removed this because it was giving a mis-leading result to the PR on the front-end. But that's the only changes I've done to templates.